packages/datastore/datastore-codegen/tests/__snapshots__/Cli.test.ts.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generate 1`] = `
+"export interface Note {
+    id: string;
+    title: string;
+    description: string;
+    comments: string[]
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type NoteCreate = Omit<Note, \\"id\\">;
+export type NoteChange =  Pick<Note, \\"id\\"> && Partial<NoteCreate>;
+
+export interface Comment {
+    id: string;
+    title: string;
+    description: string;
+    noteComment: string
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type CommentCreate = Omit<Comment, \\"id\\">;
+export type CommentChange =  Pick<Comment, \\"id\\"> && Partial<CommentCreate>;
+"
+`;
+
+exports[`generate 2`] = `
+"{
+  \\"Note\\": {
+    \\"name\\": \\"Note\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"id\\",
+    \\"properties\\": {
+      \\"id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"id\\",
+        \\"isRequired\\": true,
+        \\"index\\": true,
+        \\"primary\\": true
+      },
+      \\"title\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"title\\",
+        \\"isRequired\\": true
+      },
+      \\"description\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"description\\",
+        \\"isRequired\\": true
```

Comment:
Question. If we want to apply indexes. We might actually need some examples of overrides outside the generated source. If we override anything here it will be lost. 

I'm not sure what is the best approach here, but should be documented. 

Solution 1: copy this file outside and replicate the changes manually.
Solution 2: provide example for override in sample app.